### PR TITLE
Fix DataFlowIT issues w/ latest commits

### PR DESCRIPTION
**Pull request by onobc**
- Safe cleanup on task execution
- Deployer.<init> signature change for ActuatorOperations

### CODE REVIEW

File: spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java [hunk 1]
```diff
@@ -2354,6 +2354,34 @@ private void mixedSuccessfulFailedAndUnknownExecutions(String taskDescription, S
...
+			}
+		}
+	}
+
+	private void safeCleanupTaskExecution(Task task, long taskExecutionId) {
+		try {
+			task.cleanupTaskExecution(taskExecutionId);
+		}
+		catch (DataFlowClientException ex) {
+			if (ex.getMessage().contains("(reason: pod does not exist)") || ex.getMessage()
```
Comment (cppwfs) [inline spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java:2375]: We could probably put the exception handling code into a method.  That way we only have to change it in one place.   Thoughts?

File: spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployerTests.java [hunk 2]
```diff
@@ -624,7 +624,7 @@ public void testEnvironmentInfo() {
 		about.setVersionInfo(new VersionInfo());
 		about.getVersionInfo().setServer(new Dependency("d1", "v1", "check", "check2", "url"));
 		when(skipperClient.info()).thenReturn(about);
-		when(skipperClient.listDeployers()).thenReturn(Arrays.asList(new Deployer("d1", "t1", null)));
+		when(skipperClient.listDeployers()).thenReturn(Arrays.asList(new Deployer("d1", "t1", null, null)));
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployerTests.java:627]: Update the copyright to `2022`

File: spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java [hunk 3]
```diff
@@ -54,6 +54,7 @@
 import org.springframework.cloud.dataflow.server.repository.StreamDefinitionRepository;
 import org.springframework.cloud.dataflow.server.stream.StreamDeployerUtil;
 import org.springframework.cloud.dataflow.server.support.SkipperPackageUtils;
+import org.springframework.cloud.deployer.spi.app.ActuatorOperations;
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java:57]: Update the copyright to `2022`.

File: spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/BaseDocumentation.java [hunk 4]
```diff
@@ -36,6 +36,7 @@
 import org.springframework.cloud.dataflow.server.controller.TaskSchedulerController;
 import org.springframework.cloud.dataflow.server.service.SchedulerService;
 import org.springframework.cloud.dataflow.server.single.LocalDataflowResource;
+import org.springframework.cloud.deployer.spi.app.ActuatorOperations;
```
Comment (cppwfs) [inline spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/BaseDocumentation.java:39]: Update the copyright to `2022`.

File: spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java (hunk 1 above)
Comment (onobc) [reply spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java:2370]: Done

[the remark this answers is not in the document]
Comment (onobc) [reply spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java:2370]: This is what I did - unless I am misunderstanding you.

[the remark this answers is not in the document]
Comment (onobc) [reply spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java:2370]: Going to merge this one in @cppwfs - we can adjust anything in a follow-up commit if need be.

**Pull request merged**